weblate/templates/trans/alert/brokenbrowserurl.html

```diff
@@ -1,7 +1,7 @@
 {% load i18n %}
 <p>{% trans "This component is set up incorrectly." %}</p>
 
-<p>{% trans "The repository browser URL points to non existing locations:" %}</p>
+<p>{% trans "The repository browser URL points to a non-existing location:" %}</p>
```

Comment:
```suggestion
<p>{% trans "The repository browser URL points to non-existing locations:" %}</p>
```
On the assumption this shouldn't be "one or more non-existing locations".